package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+ <name>face_classification</name>
+ <version>1.0.0</version>
+ <description>
+ TODO!
+ </description>
+
+ <license>GPLv3</license>
+
+ <author email="miguel.oliveira.silva@tecnico.ulisboa.pt">Miguel Silva</author>
+ <maintainer email="miguel.oliveira.silva@tecnico.ulisboa.pt">Miguel Silva</maintainer>
+
+ <buildtool_depend>catkin</buildtool_depend>
+
+ <build_depend>rospy</build_depend>
+ <build_depend>message_generation</build_depend>
+
```

Comment:
Please add dependencies here and CMakeLists or make a note to add in the near future.